exseas_explorer/app.py

```diff
@@ -10,7 +10,8 @@
 from dash import Dash, dcc, html
 from dash.dependencies import Input, Output
 from dash_extensions.javascript import Namespace
-from util import (
+
+from .util import (
```

Comment:
@romatt I think this should work for you if it is not installed, but I am not 100% sure. (Otherwise mypy does not recognize this as local import.)

Reply:
Why not simply use
```
from exseas_explorer.util import (
```

Reply:
```suggestion
from exseas_explorer.util import (
```
Sure! That will work for me.